movieHWTests/MockRouter.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MockRouter.swift
+//  movieHWTests
+//
+//  Created by Александр Троицкий on 17.01.2023.
```

Comment:
```suggestion
//  Created by Александр Троицкий on 17.01.2023.
```
где форматер как у всех?